### PR TITLE
Add drf spectacular support

### PR DESCRIPTION
**Pull request by Goldziher**
This PR adds support for DRF spectacular (see issue #61 ). 

I didn't test this locally, and haven't seen any tests written for drf-yasg so I didn't write tests for this code. As such this PR is currently WIP, and I will try to find time to complete it in the near future. If someone else would like to test it or add tests that will be awesome.

Changes:
1. Added support for drf spectacular following the example of drf-yasg
2. Resolved issues reported by pytest (wrong version in assert + deprecated import statement)

### CODE REVIEW

**Comment by sondrelg** [comment]
This looks very clean. Once tests are in place, I'll be happy to merge this as is.

Before we do however, we need to add some demo code in the demo project, specifically implementing DRF-spectacular, and to build tests around that.

**Comment by Goldziher** [comment]
cheers - i'll try to find the time this weekend to finish this.

**Comment by sondrelg** [comment]
I've just pushed through version 2.0.0, with a revised setup for loader classes. I actually think it will make the implementation of a DRF-spectacular loader class even cleaner than before. 

I've also upgraded the demo project to Django 3.1, so unfortunately this might cause some conflicts.

An example of the revised drf-yasg loader class can be found [here](https://github.com/snok/django-swagger-tester/blob/master/django_swagger_tester/loaders.py#L344). Let me know if you need any help.

**Comment by Goldziher** [comment]
Ok, I updated the PR and added some tests along the lines you had in place 😄 . So as far as im concerned this one is finished!

**Comment by sondrelg** [comment]
Great! I'll review it today 👍

File: django_swagger_tester/loaders.py [hunk 1]
```diff
@@ -406,7 +411,66 @@ def get_route(self, route: str) -> Route:
...
+
+    def get_path_prefix(self) -> str:
+        """
+        Returns the drf_spectacular specified path prefix.
+        """
+        from drf_spectacular.settings import spectacular_settings
+
+        return spectacular_settings.SCHEMA_PATH_PREFIX
+
+    def get_route(self, route: str) -> str:
```
Comment (sondrelg) [inline django_swagger_tester/loaders.py:460]:
```suggestion
    def get_route(self, route: str) -> Route:
```

**Review by sondrelg — APPROVED** [review]
Looks good to me :)

**Comment by sondrelg** [comment]
I'm aiming to merge #74 this week and update the docs, for which I could also add some info about this implementation.

If it's OK with you, I'll wait to push this to pypi until that's done, and it will be released as a part of v2.1.0.

**Comment by Goldziher** [comment]
sure thing, sounds good. Cheers :)

**Pull request merged**